script/fetch-issues.js

```diff
@@ -14,14 +14,13 @@ let earliestDate = process.env.EARLIEST_DATE || '2017-01-01T00:00:00Z';
 
 if (isDeployPreview) {
   const earliestDateForPr = process.env.EARLIEST_DATE_PR ||
-    '2018-04-01T00:00:00Z';
-  if (getTime(earliestDate) < getTime(earliestDateForPr)) {
+    moment().subtract(3, 'months').format('YYYY-MM-DDThh:mm:ssZ');
     earliestDate = earliestDateForPr;
-    console.log('To speed up netlify pr build, set earliest date threshold',
-      earliestDate);
   }
```

Comment:
Obviously, this will cause a syntax error. Please run it locally first, at least you should make sure it does not cause any linting error/compilation error before you push.

Reply:
Really sorry about it. I missed the linting. Wouldn't hurry to push now-onward.